### PR TITLE
Add a contribute.json file.

### PR DESCRIPTION
**Pull request by akatsoulas**

### CODE REVIEW

**Comment by akatsoulas** [comment]
@johngian r?

File: contribute.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+    "name": "feedthefox",
+    "description": "FirefoxOS Participation Hub",
```
Comment (comzeradd) [inline contribute.json]: I think Firefox OS should be two words.

File: contribute.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    "participate": {
+        "irc": "irc://irc.mozilla.org/#partinfra"
+    },
+    "repository": {
+        "url": "https://github.com/mozilla/feedthefox",
+        "license": "MPL2"
+    },
+    "bugs": {
+        "list": "https://github.com/mozilla/feedthefox/issues",
+        "report": "https://github.com/mozilla/feedthefox/issues"
```
Comment (comzeradd) [inline contribute.json]: You could use the [new](https://github.com/mozilla/feedthefox/issues/new) url for reporting.

File: contribute.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        "dev": "https://feedthefox-dev.pass.mozilla.community",
+        "prod": "https://firefoxos.mozilla.community"
+    },
+    "keywords": [
+        "django",
+        "python",
+        "docker",
+        "jquery",
+        "html",
+        "css"
```
Comment (comzeradd) [inline contribute.json]: Add javascript too :)

**Comment by akatsoulas** [comment]
@comzeradd pr updated

**Comment by comzeradd** [comment]
Tested locally and code looks good. r+

**Pull request merged**